component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -586,6 +605,39 @@ public static void redirectToEnableTOTPReqPage(HttpServletRequest request, HttpS
         }
     }
 
+    /**
+     * Redirect the enableTOTP request page.
```

Comment:
Lets improve the method comment a bit more to include what happens here. It was a bit hard to understand whats going on here

Reply:
fixed with fce8e8503344493ed0136f775e2bd5cd7761e345